src/views/Activity.vue

```diff
@@ -30,127 +30,120 @@ div
 
   aw-periodusage(:periodusage_arr="daily_activity", :host="host")
 
-  ul.nav.nav-tabs
-    li.nav-item.aw-nav-item
-      a.nav-link.aw-nav-link(v-on:click="view = 'summary'" v-bind:class="{ active: view=='summary' }")
-        h5 Summary
-    li.nav-item.aw-nav-item
-      a.nav-link.aw-nav-link(v-on:click="view = 'window'" v-bind:class="{ active: view=='window' }")
-        h5 Window
-    li.nav-item.aw-nav-item
-      a.nav-link.aw-nav-link(v-on:click="view = 'browser'" v-bind:class="{ active: view=='browser' }")
-        h5.active-h5 Browser
-    li.nav-item.aw-nav-item
-      a.nav-link.aw-nav-link(v-on:click="view = 'editor'" v-bind:class="{ active: view=='editor' }")
-        h5 Editor
-  br
-
-  div.row(v-show="view == 'summary'")
-    div.col-md-4
-      h5 Top Applications
-      aw-summary(:fields="top_apps", :namefunc="top_apps_namefunc", :colorfunc="top_apps_colorfunc")
-      b-button(size="sm", variant="outline-secondary", :disabled="top_apps.length < top_apps_count", v-on:click="top_apps_count += 5; queryWindows()")
+  // Nav tabs
+  ul.nav.nav-tabs.nav-pills.lead.mb-3.mt-4
```

Comment:
The reason why I went with custom CSS was because the normal tabs were not very visible and the pills looked more like buttons than tabs as well as are very blue (while the rest of the buttons in the UI are grey/black) so it it didn't really fit with the design IMO.